include/fi_atom.h

```diff
@@ -53,7 +53,7 @@ extern "C" {
 
 
 #if ENABLE_DEBUG
-#define ATOMIC_DEF_INIT int is_initialized
+#define ATOMIC_DEF_INIT int is_initialized;
```

Comment:
I updated PR: added support of built-in GCC (and ICC) atomics, and windows implementation is based on built-in atomics too